platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -485,11 +485,13 @@ data:
   #
   #
   type: h2
-  ## @param: url
-  ## @type: string
-  ## Location of the database
-  #
-  url: jdbc:h2:mem:anchor-platform
+
+  ## The hostname and port of the database server.
+  server:
+
+  ## Name of the database.
+  database:
```

Comment:
What do you think about using `dbName` here?

Reply:
Not a big deal, just an idea. Its not totally clear that you're looking for the database name when I use the `DATA_DATABASE` environment variable.

Reply:
from Postgres doc: https://jdbc.postgresql.org/documentation/use/

They use `database` as the database name. I think we can keep database unless you feel strong about this.

Reply:
Nope, thats fine.